fibers.cpp

```diff
@@ -291,16 +296,16 @@ void Fibers::unblock(int fiberIndex)
 void Fibers::end()
 	{
 	LOG("end " << curFiberIndex());
-	verify(!inMain());
-	cur->status = Fiber::ENDED;
+	verify(!inMain());	

```

Comment:
whitespace on end of line (might want to configure editor to remove this automatically)